.gitlab-ci.yml

```diff
@@ -238,6 +254,8 @@ init_workspace:
   tags:
     - mender-qa-slave
   before_script:
+    # Default value, will later be overwritten if successful
+    - echo "failure" > ${HOME}/JOB_RESULT.txt
```

Comment:
Still wrong, it should go after line 335.

Alternatively, and probably better, move that workspace untar logic to the top of before_script

Reply:
Argh, even `HOME` is being redefined... Alright, how about simply hardcoding this to `/JOB_RESULT.txt` then? The thing is that I really want this to be independent from other steps we do in the recipe, exactly so that we can catch errors we make in such sections.

Reply:
`/JOB_RESULT.txt` is a good place :+1:

Reply:
Fixed.